bluemira/radiation_transport/radiation_profile.py

```diff
@@ -1404,7 +1494,7 @@ def calculate_sol_distribution(self, firstwall_geom: Grid):
                 flux_tubes=getattr(self, f"flux_tubes_{side}_{low_up}"),
                 x_strike=getattr(self, f"x_strike_{side}"),
                 z_strike=getattr(self, f"z_strike_{side}"),
-                main_ext=None,
+                main_ext=3,
```

Comment:
whats this mean?

Reply:
@je-cook I parametrised `main_ext` as it should be a user choice. Does it work better?